libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -134,11 +133,10 @@ export class TypingsGenerator {
     let checkFilePaths: boolean = true;
     if (!relativeFilePaths?.length) {
       checkFilePaths = false; // Don't check file paths if we generate them
-      relativeFilePaths = await LegacyAdapters.convertCallbackToPromise(glob, this.inputFileGlob, {
+      relativeFilePaths = await glob(this.inputFileGlob, {
         cwd: this.sourceFolderPath,
-        ignore: this.ignoredFileGlobs,
-        nosort: true,
-        nodir: true
+        ignore: this.ignoredFileGlobs as string[],
```

Comment:
Hmm, should we allocate a new array for this? It's unlikely it'll get changed so this is not really a concern of mine, but more asking for my own knowledge of what the best design here would be.

Reply:
This is really just a case of poor typings. I expect `ReadonlyArray<string>` is probably the correct input type.

Reply:
Should we then change the signature on our side?